Source/SpeechToTextV1/SpeechToTextSession.swift

```diff
@@ -69,12 +78,28 @@ public class SpeechToTextSession {
         set { socket.onDisconnect = newValue }
     }
     
-    private let socket: SpeechToTextSocket
+    private lazy var socket: SpeechToTextSocket = {
```

Comment:
I think this is the first `lazy var` we've used. It's neat to get some intuition behind its practical usage.